src/chains.jl

```diff
@@ -375,7 +375,6 @@ max_datetime(ts) = maximum(to_datetime_vec(ts))
 
 # does not specialize on `typeof(T)`
 function missing_datetime(T::Type)
-    @warn "timestamp of type $(T) unknown"
     return missing
 end
```

Comment:
It seems this method is only called by `to_datetime` above? In that case, could we just remove it conpletely and directly return `missing` in `to_datetime`?

Reply:
We'd have to pick a point to intercept. Currently, I now have `to_datetime_vec` return `missing`, and then `(min/max)_datetime` forward `missing`s.

This is because of calls like
```julia
min_start(c::Chains) = min_datetime(start_times(c))
```
`start_times` will return `missing`, and then `min_datetime` must handle it.

We could also change the definition of `min_start` to check whether `info` contains the appropriate field.